### ISSUE TITLE
Input guidance is not accessible via keyboard or screen readers

### ISSUE THREAD

**Issue by jamesjacobs** [jadu/pulsar#894]
Currently, there is no way to access input "guidance" without a mouse click. Also, screen readers do not read guidance text.

I have a few ideas on how to resolve this so I've assigned to me for the time being.

**Comment by jamesjacobs** [comment]
Guidance popovers have been deprecated
